### PR TITLE
UML-2779: Adding new event codes for type of LPA

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-2779

Fixes UML-2779

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have added relevant logging with appropriate levels to my code
* [X] New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
~* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant~
~* [ ] I have added tests to prove my work~
~* [ ] I have added welsh translation tags and updated translation files~
~* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found~
~* [ ] I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made~
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by SeemaMenon** [comment]
Screenshot 2023-02-15 at 14 42 24
Screenshot 2023-02-15 at 15 19 09

File: service-front/app/src/Actor/src/Handler/CheckLpaHandler.php [hunk 1]
```diff
@@ -84,9 +86,9 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         $this->user     = $this->getUser($request);
         $this->identity = !is_null($this->user) ? $this->user->getIdentity() : null;
 
-        $activation_key  = $this->state($request)->activationKey;
-        $referenceNumber = $this->state($request)->lpaReferenceNumber;
-        $dob             = $this->state($request)->dateOfBirth?->format('Y-m-d');
+        $activation_key     = $this->state($request)->activationKey;
```
Comment (cooperaj) [inline service-front/app/src/Actor/src/Handler/CheckLpaHandler.php:89]: Theres a lot of this extra spacing being added. I'm not sure what settings are doing this. The general rule for consecutive assignment blocks like this is that the spacing should be the same as the longest variable name. So what this code originally was.

**Review by cooperaj — CHANGES_REQUESTED** [review]
Just weird extra spaces throughout. Otherwise looks great.

File: service-front/app/src/Actor/src/Handler/CheckLpaHandler.php (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/src/Handler/CheckLpaHandler.php:89]: Something bad and mysterious on my machine ...my bad ! Corrected manually  @cooperaj

**Review by cooperaj — APPROVED** [review]

**Pull request merged**